### ISSUE TITLE
Cannot read property 'default' of undefined: if style/css loaders used

### ISSUE THREAD

**Issue by develar** [s-panferov/awesome-typescript-loader#20]
My config: https://github.com/develar/cloud-intellij/blob/0aec6770cf1996ba970c2f58dc8d7e8791c2ab09/web-client/webpack.config.js

Works fine if ExtractTextPlugin/style-loader are not used.

```/Users/develar/Documents/cloud-intellij/web-client/node_modules/awesome-typescript-loader/dist/index.js:64
        var instance = compilation.compiler._tsInstances[instanceName];
                                                        ^
TypeError: Cannot read property 'default' of undefined
    at Tapable.<anonymous> (/Users/develar/Documents/cloud-intellij/web-client/node_modules/awesome-typescript-loader/dist/index.js:64:57)
    at Tapable.next (/usr/local/lib/node_modules/webpack-dev-server/node_modules/webpack/node_modules/tapable/lib/Tapable.js:69:14)
    at Tapable.<anonymous> (/usr/local/lib/node_modules/webpack-dev-server/node_modules/webpack/lib/CachePlugin.js:40:4)
    at Tapable.applyPluginsAsync (/usr/local/lib/node_modules/webpack-dev-server/node_modules/webpack/node_modules/tapable/lib/Tapable.js:71:13)
    at Tapable.<anonymous> (/usr/local/lib/node_modules/webpack-dev-server/node_modules/webpack/lib/Compiler.js:398:9)
    at Tapable.<anonymous> (/usr/local/lib/node_modules/webpack-dev-server/node_modules/webpack/lib/Compilation.js:561:13)
    at Tapable.applyPluginsAsync (/usr/local/lib/node_modules/webpack-dev-server/node_modules/webpack/node_modules/tapable/lib/Tapable.js:60:69)
    at Tapable.<anonymous> (/usr/local/lib/node_modules/webpack-dev-server/node_modules/webpack/lib/Compilation.js:556:10)
    at Tapable.applyPluginsAsync (/usr/local/lib/node_modules/webpack-dev-server/node_modules/webpack/node_modules/tapable/lib/Tapable.js:60:69)
    at Tapable.<anonymous> (/usr/local/lib/node_modules/webpack-dev-server/node_modules/webpack/lib/Compilation.js:551:9)
    at Tapable.applyPluginsAsync (/usr/local/lib/node_modules/webpack-dev-server/node_modules/webpack/node_modules/tapable/lib/Tapable.js:60:69)
    at Tapable.<anonymous> (/usr/local/lib/node_modules/webpack-dev-server/node_modules/webpack/lib/Compilation.js:547:8)
    at Tapable.applyPluginsAsync (/usr/local/lib/node_modules/webpack-dev-server/node_modules/webpack/node_modules/tapable/lib/Tapable.js:60:69)
    at Tapable.seal (/usr/local/lib/node_modules/webpack-dev-server/node_modules/webpack/lib/Compilation.js:511:7)
    at Tapable.<anonymous> (/usr/local/lib/node_modules/webpack-dev-server/node_modules/webpack/lib/Compiler.js:395:15)
    at /usr/local/lib/node_modules/webpack-dev-server/node_modules/webpack/node_modules/tapable/lib/Tapable.js:103:11
    at Tapable.<anonymous> (/usr/local/lib/node_modules/webpack-dev-server/node_modules/webpack/lib/Compilation.js:437:10)
    at /usr/local/lib/node_modules/webpack-dev-server/node_modules/webpack/lib/Compilation.js:409:12
    at /usr/local/lib/node_modules/webpack-dev-server/node_modules/webpack/lib/Compilation.js:324:10
    at done (/usr/local/lib/node_modules/webpack-dev-server/node_modules/webpack/node_modules/async/lib/async.js:132:19)
    at /usr/local/lib/node_modules/webpack-dev-server/node_modules/webpack/node_modules/async/lib/async.js:32:16
    at process._tickCallback (node.js:355:11)```

**Comment by s-panferov** [comment]
@develar 

Fixed in v0.5.0. Please see corresponding [example](https://github.com/s-panferov/awesome-typescript-loader/tree/master/examples/extract-text)

**Comment by develar** [comment]
Thanks a lot for quick fix.
